tests/Mailer/SymfonyAuthCodeMailerTest.php

```diff
@@ -59,6 +59,44 @@ public function sendAuthCode_hasAuthCode_sendEmail(): void
         $this->mailer->sendAuthCode($user);
     }
 
+    /**
+     * @test
+     */
+    public function sendAuthCode_hasAuthCode_sendEmail_withoutSender(): void
+    {
+        $mailer = new SymfonyAuthCodeMailer($this->symfonyMailer, null, null);
+
+        // Stub the user object
+        $user = $this->createMock(TwoFactorInterface::class);
+        $user
+            ->expects($this->any())
+            ->method('getEmailAuthRecipient')
+            ->willReturn('recipient@example.com');
+        $user
+            ->expects($this->any())
+            ->method('getEmailAuthCode')
+            ->willReturn('1234');
+
+        $messageValidator = function ($mail) {
+            /** @var Email $mail */
+            $this->assertInstanceOf(Email::class, $mail);
+            $this->assertEquals('recipient@example.com', current($mail->getTo())->getAddress());
+            $this->assertNull(current($mail->getFrom()));
```

Comment:
Seems to return `false` instead of `null`, `current()` of  empty array is `false`

Reply:
done :)